### ISSUE TITLE
Cross training between different tasks

### ISSUE THREAD

**Issue by yellowjs0304** [ultralytics/ultralytics#2091]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi, Could you help me if you have any idea of this task?

First of all, i have large benchmark dataset and small specific custom dataset. they have similar concept, but the domain is little bit different.
I wanna "pre-train the benchmark set and finetuning it with my custom data", but, it has really large time and cost for training benchmark set.

I already have pre-trained segment model w large benchmark dataset.
Can i use this model for pose detection training w custom data? (like visual extractor)

And if it can be, Where should i modify for this task?

**Comment by Laughing-q** [comment]
@yellowjs0304 you mean you want to use the pre-trained segment model as initialized weight and freeze the backbone part? and train the rest part of the whole network?

**Comment by yellowjs0304** [comment]
@Laughing-q Thank you for reply.
You're right. I want to use it as pre-trained model.
But i don't want to freeze all the backbone part at all.  
Just applying a small lr and finetuning whole network.

**Comment by Laughing-q** [comment]
@yellowjs0304 I think `pretrained` should work to you, i.e
for cli:
```bash
yolo pose train model=yolov8n-pose.yaml pretrained=your_segment_model.pt
```
for python:
```python
model = YOLO("yolov8n-pose.yaml").load("segment_model")
```

**Comment by yellowjs0304** [comment]
@Laughing-q okay, Thank you for helping me.

Last one, Do you remember the range of box loss and kpts loss, when the model(shared weight) is optimized?

My best model has only 0.75(bbox mAP50-95) 0.94(kpts mAP50-95) in my custom dataset(train 865, val: 299, test: 300) during 786 Epochs.(1000 epochs but early stopped on Epoch 786, patience = default value)
Also the loss is like below.

```
train/box_loss	        train/pose_loss	        train/kobj_loss	         train/cls_loss
  0.30482	                0.02628	                0.03471	                0.24425
```

I think it's a little bit higher than my expectation. How do you think about this train log...
![image](https://user-images.githubusercontent.com/30281608/232709804-442056f3-e0f5-4ef9-89b6-fbbc867f26a9.png)

**Comment by glenn-jocher** [comment]
@yellowjs0304 The YOLOv8 model is optimized by minimizing several loss functions such as box loss, pose loss, kobj loss, and classification loss. The optimal range of each loss function may vary depending on the image dataset, architecture, and hyperparameters used. Your model has bbox mAP50-95 of 0.75 and kpts mAP50-95 of 0.94, which is not bad. However, to improve your model further, you can try fine-tuning and experimenting with different hyperparameters such as learning rate, batch size, and also try to increase the dataset size. You can also refer to the YOLOv8 documentation for more information or ask for further guidance on the Ultralytics YOLOv8 GitHub Issues.

**Comment by yellowjs0304** [comment]
Thanks for your help.

**Comment by glenn-jocher** [comment]
@yellowjs0304 okay, understood. If you have any further questions, feel free to ask. I'll do my best to help you. Also, you can visit the Ultralytics YOLOv8 documentation and the GitHub Issues for additional information on YOLOv8.
